src/components/ReceivedFiles.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, Fragment } from 'react';
+import { ReceivedFile } from '../constants/Types';
+
+type ReceivedFilesProps = {
+    receivedFiles: ReceivedFile[];
+};
+
+const ReceivedFiles: FC<ReceivedFilesProps> = (props: ReceivedFilesProps) => {
+    const receivedFiles = props.receivedFiles.map((receivedFile: ReceivedFile) => {
+        return (
+            <a href={receivedFile.anchorDownloadHref} download={receivedFile.fileName}>
```

Comment:
Small thing but I think we need to add a `key` param for `.map` renders.

`props.receivedFiles.map((receivedFile: ReceivedFile, idx: number) => {
     return (
            <a key={idx} ... >
     ...
`